rcl/test/rcl/service_fixture.cpp

```diff
@@ -37,33 +37,37 @@ wait_for_service_to_be_ready(
   rcl_wait_set_t wait_set = rcl_get_zero_initialized_wait_set();
   rcl_ret_t ret = rcl_wait_set_init(&wait_set, 0, 0, 0, 0, 1, rcl_get_default_allocator());
   if (ret != RCL_RET_OK) {
-    RCUTILS_LOG_ERROR("Error in wait set init: %s", rcl_get_error_string_safe())
+    RCUTILS_LOG_ERROR_NAMED(
+      ROS_PACKAGE_NAME, "Error in wait set init: %s", rcl_get_error_string_safe());
     return false;
   }
   auto wait_set_exit = make_scope_exit(
     [&wait_set]() {
       if (rcl_wait_set_fini(&wait_set) != RCL_RET_OK) {
-        RCUTILS_LOG_ERROR("Error in wait set fini: %s", rcl_get_error_string_safe())
+        RCUTILS_LOG_ERROR_NAMED(
+          ROS_PACKAGE_NAME, "Error in wait set fini: %s", rcl_get_error_string_safe());
         throw std::runtime_error("error waiting for service to be ready");
       }
     });
   size_t iteration = 0;
   do {
     ++iteration;
     if (rcl_wait_set_clear_services(&wait_set) != RCL_RET_OK) {
-      RCUTILS_LOG_ERROR("Error in wait_set_clear_services: %s", rcl_get_error_string_safe())
+      RCUTILS_LOG_ERROR_NAMED(
+        ROS_PACKAGE_NAME, "Error in wait_set_clear_services: %s", rcl_get_error_string_safe());
       return false;
     }
     if (rcl_wait_set_add_service(&wait_set, service) != RCL_RET_OK) {
-      RCUTILS_LOG_ERROR("Error in wait_set_add_service: %s", rcl_get_error_string_safe())
+      RCUTILS_LOG_ERROR_NAMED(
+        ROS_PACKAGE_NAME, "Error in wait_set_add_service: %s", rcl_get_error_string_safe());
```

Comment:
Nit: added trailing semi-colon (same multiple times in this file)

Reply:
oops, removed in decb56a2c7dcc09e951f54a1a5a32e25d3283afa